tests/unit/model/designation-test.js

```diff
@@ -54,3 +54,191 @@ test('Relationships with other models', function(assert){
   assert.equal(relationshipOrdersPackage.key, 'ordersPackages');
   assert.equal(relationshipOrdersPackage.kind, 'hasMany');
 });
+
+test('check isLocalOrder', function(assert){
+  const model = this.subject({detailType: 'LocalOrder'});
```

Comment:
@viveksriram1 , what is the need of declaring this as a constant? as it is only used once and there aren't any operations which will update the model object.